server/health-check.ts

```diff
@@ -1,6 +1,7 @@
 import { execSync } from "child_process";
 import { readFileSync, existsSync, statSync } from "fs";
 import { AGENT_SERVER_PORT, LOCAL_AGENT_INSTANCE_ID } from "./constants";
+import { getVersion } from "../worker/utils";
```

Comment:
![image](https://github.com/user-attachments/assets/1b0881c4-16b4-451c-873f-dfb8fb30d653)

Fix this build error by never importing a method from worker into server. Instead, move the method to server